hummingbird.py

```diff
@@ -65,13 +65,12 @@ def __init__(self):
             addresses = get_MAC(line=stdin.readline())
             self.waiting_for_input = False
             for address in addresses:
-                
                 ## Pings Hummingbird Django server. If use_cache is set to True, use self.local_payload_cache as a cache.
                 if config.use_cache:
                     if address.lower() in self.local_payload_cache:
                         # payload is a tuple of address and last_checked_datetime
                         payload = self.local_payload_cache[address.lower()]
-                        # Set a 60 second cache             
+                        # Set a cache for cache_time_seconds             
```

Comment:
I like that you changed the comment since the time depends on what is in the configuration file, but what you had is probably more intuitive when glancing over the code. Here's how we can compromise between the two:
```python
# Set a cache for cache_time_seconds (defaults to 60 seconds)
```